src/probnum/diffeq/ode/ivp_examples.py

```diff
@@ -495,3 +496,65 @@ def threebody_rhs(t, y, params):
     y1p = y1 + 2.0 * y2_dot - mp * (y1 + mu) / d1 - mu * (y1 - mp) / d2
     y2p = y2 - 2.0 * y1_dot - mp * y2 / d1 - mu * y2 / d2
     return np.array([y1_dot, y2_dot, y1p, y2p])
+
+
+def lorenz(timespan, initrv, params=(10, 28, 8 / 3)):
```

Comment:
You might want to set the default parameters as `float`s, but I think it will make no difference.

Reply:
i.e., `params=(10.0, 28.0, 8.0 / 3.0)`